libnopegl/src/node_text.c

```diff
@@ -283,60 +276,55 @@ static int update_character_geometries(struct ngl_node *node)
             corner[2] + width[2] * chr->x + height[2] * chr->y,
         };
 
-        /* deduce quad vertices from chr_{width,height,corner} */
-        const float chr_vertices[] = {
-            C(0),               C(1),               C(2),
-            C(0) + W(0),        C(1) + W(1),        C(2) + W(2),
-            C(0) + H(0),        C(1) + H(1),        C(2) + H(2),
-            C(0) + H(0) + W(0), C(1) + H(1) + W(1), C(2) + H(2) + W(2),
```

Comment:
You can also remove C(), W() and H().

Reply:
Oh yeah I did it in another branch and forgot to import the chunk, thanks, done